.github/workflows/integration.yml

```diff
@@ -0,0 +1,55 @@
+name: hashicorp/waypoint-hzn/integration
+on:
+  push:
+    # TODO: do we want to push a new image everytime we push to main (including this new PR)?
+    branches:
+      # - main
```

Comment:
Confirming that we want a new image to be pushed on every push to main?

Reply:
I imagine we just remove `integration.yml` here